playbook-website/app/javascript/components/Website/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react"
+import { Layout, Icon } from "playbook-ui"
+import Sidebar from "./src/layouts/Sidebar"
+import LayoutRight from "./src/layouts/LayoutRight"
+
+function App() {
+  const [kits, setKits] = useState([])
+  const [dark, setDark] = useState(false)
+  const [isMobile, setIsMobile] = useState(window.innerWidth < 768)
+
+  useEffect(() => {
+    fetch("/beta/kits.json")
+      .then((response) => response.json())
+      .then((data) => {
+        setKits(data.kits)
+        setDark(data.dark)
+      })
+      .catch((error) => {
+        console.log(error)
+      })
+
+    window.addEventListener("resize", () => {
+      setIsMobile(window.innerWidth < 768)
```

Comment:
```tsx
window.innerWidth < 768
```

This part really should be a constant so that it is DRYer

Reply:
Can we import them? I see [we have an export](https://github.com/powerhome/playbook/blob/master/playbook/app/pb_kits/playbook/tokens/exports/_screen_sizes.scss)

`$screen-md-min`